JavaScript/JavaScriptSDK/snippet.js

```diff
@@ -30,8 +30,12 @@
         appInsights[name] = function() {
             // Capture the original arguments passed to the method
             var originalArguments = arguments;
+            if (name.indexOf(trackPage) >= 0) {
+                [].push.call(originalArguments, new Date().getTime());
```

Comment:
Does it work when a user calls `trackPageView` without `name` parameter?  Is new Date used as the name?